app/assets/stylesheets/components/_calendars.scss

```diff
@@ -1,5 +1,21 @@
 $calendar-today-background: $color-white;
 
+.guiders-appointments-calendar {
+  .fc-resource-cell {
+    height: 150px;
+    white-space: nowrap;
+
+    div {
+      transform: translateY(120px) rotate(-90deg);
```

Comment:
I think this is going to need some looking at as I get a weird layout:

<img width="1177" alt="screen shot 2016-10-28 at 14 18 38" src="https://cloud.githubusercontent.com/assets/295469/19807961/62956b16-9d1a-11e6-8dde-45de83331693.png">

cc @Guntrisoft